lib/iris/cube.py

```diff
@@ -1919,6 +1957,73 @@ def coord_system(self, spec=None):
 
         return result
 
+    def _a_meshcoord(self):
```

Comment:
@pp-mo Could we rename this just to be just `_meshcoord`? I think that's fine. Or even `_any_meshcord`?

Also add a slim doc-string for developers i.e., `returns any MeshCoord that's available`